libcst/_parser/_types/config.py

```diff
@@ -63,21 +63,47 @@ def __repr__(self) -> str:
 @add_slots
 @dataclass(frozen=True)
 class PartialParserConfig:
-    """
+    r"""
     An optional object that can be supplied to the parser entrypoints (e.g.
     `parse_module`) to configure the parser.
```

Comment:
Not related to this change but can you convert the `parse_module` as a link to the actual function?